### DISCUSSION TITLE
530 Authentication failed error, keeps timing out

### DISCUSSION THREAD

**Discussion by JoWhite76** [iterate-ch/cyberduck#16643] [Q&A]
I'm on a MAC and trying to use Cyberduck, I've never had any issues until now.
530 Authentication failed error, keeps timing out. Any suggestions would be appreciated.
![Screenshot 2024-12-10 at 08 13 52](https://github.com/user-attachments/assets/9b7fcbad-cfb7-47fd-90a9-76a31f4ee307)

**Comment by dkocher** [comment]
Contact your hosting service provider to make sure the server is up and running and accepting requests.
